cmd/kube-apiserver/app/server.go

```diff
@@ -208,7 +212,29 @@ func Run(s *options.APIServer) error {
 	if err != nil {
 		glog.Errorf("Failed to create clientset: %v", err)
 	}
-	admissionController := admission.NewFromPlugins(client, admissionControlPluginNames, s.AdmissionControlConfigFile)
```

Comment:
Looks like you still want this method, but you want it to return a `[]admission.Interface`, right?